impl/src/main/java/com/okta/sdk/impl/oauth2/OAuth2ErrorResponse.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.oauth2;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OAuth2ErrorResponse {
```

Comment:
This should probably extend `Error` and expse the error properties in a standard way

Reply:
This error object exists for the sole reason of parsing the json response body from OAuth auth server and maybe propagate it via exception objects. I dont see a need why this should implement `com.okta.sdk.error.Error` interface.

Reply:
Got rid of this class. Hence closing this comment.